model/registry/http.yaml

```
@@ -133,3 +133,15 @@ groups:
           MUST NOT be populated when this is not supported by the HTTP server framework as the route attribute should have low-cardinality and the URI path can NOT substitute it.
 
           SHOULD include the [application root](/docs/http/http-spans.md#http-server-definitions) if there is one.
+      - id: connection.state
+        type:
+          members:
```

nit:

```suggestion
          allow_custom_values: true
          members:
```

(we should really change the default in the build-tools or even remove this flag)

---

https://github.com/open-telemetry/build-tools/issues/294